text-utils/col.c

```diff
@@ -154,9 +150,10 @@ static void __attribute__((__noreturn__)) usage(void)
 	exit(EXIT_SUCCESS);
 }
 
-static void __attribute__((__noreturn__)) wrerr(void)
+static inline void col_putchar(wchar_t ch)
 {
-	errx(EXIT_FAILURE, _("write error"));
+	if (putwchar(ch) == WEOF)
+		errx(EXIT_FAILURE, _("write error"));
```

Comment:
Would be better to use _("write failed") as we have in other tools? Would be possible to use err() to get  details about the error?

Reply:
Text is fixed in 3375fd9ccf, and I checked  the err() is used not the errx().